libs/cdk/jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   displayName: 'cdk',
   preset: '../../jest.preset.js',
   coverageReporters: ['lcov', 'cobertura'],
-  coverageDirectory: '../../docs/test-coverage/cdk',
+  coverageDirectory: '../../dist/apps/docs/test-coverage/cdk',
```

Comment:
The build output directory for the `docs` project is now `dist/apps/docs` instead of the raw files in `docs`.